workflows/methylation/methylation-cohort.wdl

```diff
@@ -15,10 +15,12 @@ workflow methylation_cohort {
 
     parameter_meta {
         unfiltered_normalized_beta: "Array of unfiltered normalized beta values for each sample"
+        num_probes: "Number of probes to retain after filtering"
```

Comment:
This feels like it could have a little more information. Maybe keep this text as the description, and then add a help key?